BE/OwnershipAndControl/Executives.rdf

```diff
@@ -520,6 +551,14 @@
 		<skos:definition>some party that has some defined responsibility with respect to the formal organization</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;hasSigningAuthorityFor">
+		<rdfs:subPropertyOf rdf:resource="&fibo-be-oac-exec;isAuthorizedBy"/>
+		<rdfs:label>has signing authority for</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-oac-exec;Signatory"/>
+		<owl:inverseOf rdf:resource="&fibo-be-oac-exec;designatesSignatory"/>
+		<skos:definition>indicates the party for which a signatory has the ability to sign agreements, access accounts and perform related tasks</skos:definition>
+	</owl:ObjectProperty>
+	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;hasVestedInIt">
```

Comment:
poorly worded and defined, is it needed?

Reply:
Fixed the wording and this is important for a number of banking role-related use cases - for power of attorney, for signing authority on an account, etc.